### DISCUSSION TITLE
Is there a way to replace all the `\:`s in the className with `_` ?

### DISCUSSION THREAD

**Discussion by 2234839** [tailwindlabs/tailwindcss#3402] [Help] [answered]
Because restricted css files like wxss cannot accept the presence of `\`  ![图片](https://user-images.githubusercontent.com/28727933/105139722-966a3500-5b31-11eb-8c0f-fe26f857dfd0.png)

**Comment by adamwathan** [comment, marked as answer]
Hey! You can use the [separator config option](https://tailwindcss.com/docs/configuration#separator) to specify a different character.
